parser/parser/cc/grammars/typedruby.ypp

```diff
@@ -486,7 +486,7 @@ void parser::error(const ruby_parser::location &lloc, const std::string &msg) {
 
 	driver.diagnostics.emplace_back(
 		dlevel::ERROR, dclass::UnexpectedToken,
-		diagnostic::range(driver.lex.last_token_s, driver.lex.last_token_e),
+		diagnostic::range(lloc.begin, lloc.end),
```

Comment:
This was the only use of `last_token_s` and `last_token_e`, and the arity of `parser::error` changes when you've passed the `%define api.location.type` to bison to include `lloc`, which has the `begin` and `end` set to the same thing that last_token_{s,e} had.